internal/ref/ref.go

```diff
@@ -26,3 +26,18 @@ func ParseDataPath(s string) (ast.Ref, error) {
 
 	return path.Ref(ast.DefaultRootDocument), nil
 }
+
+// ArrayPath will take an ast.Array and build an ast.Ref using the ast.Terms in the Array
+func ArrayPath(a *ast.Array) ast.Ref {
+	var ref ast.Ref
+
+	err := a.Iter(func(term *ast.Term) error {
+		ref = append(ref, term)
+		return nil
+	})
+	if err != nil {
+		return ast.Ref{}
+	}
```

Comment:
nit: use `a.Foreach` instead so that the error condition doesn't exist (technically that if err != nil branch is not reachable)

Reply:
Also fixed this in 5cc4668f9cbd41f90df51e9d1d1031afdcb5bf7f 🤞